packages/nx-sonarqube/src/executors/scan/utils/utils.ts

```diff
@@ -8,12 +8,13 @@ import {
   logger,
   ProjectGraph,
   readCachedProjectGraph,
+  readJsonFile,
 } from '@nx/devkit';
-import { readFileSync } from 'fs';
 import { tsquery } from '@phenomnomnominal/tsquery';
 import { execSync } from 'child_process';
 import * as sonarQubeScanner from 'sonarqube-scanner';
 import { TargetConfiguration } from 'nx/src/config/workspace-json-project-json';
+import { readFileSync } from 'fs';
```

Comment:
dont need this or in the unit tests

Reply:
what nx equivalent function should I use to read files ?